src/datatypes/compiler-conditional.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
       return compiler.wrapCode(code, args)
     }],
     'option': ['parametrizable', (compiler, type) => {
-      let code = 'if (value !== null) {\n'
+      let code = 'if (value) {\n'
```

Comment:
This should be != null, it's not equivalent

Reply:
done